gtests/net/packetdrill/run_all.py

```diff
@@ -254,6 +260,8 @@ def ParseArgs():
   """Parse commandline arguments."""
   args = argparse.ArgumentParser()
   args.add_argument('path', default='.', nargs='?')
+  args.add_argument('-c', '--capture', metavar='DIR',
+                    help='capture packets in the specified directory')
```

Comment:
nit: capture packets and store as .pcap files in the specified directory, one per test.